src/Template/Element/Flash/default.ctp

```diff
@@ -1,6 +1,6 @@
 <?php
 $class = array_unique((array)$params['class']);
-$message = h($message);
+$message = isset($params['escape']) && $params['escape'] === false ? $message : h($message);
```

Comment:
No Problem! Thanks for checking. You had me second guessing myself for a moment, I was thinking of the thousands of ternary statements I have used in the past that looked just like that!